src/components/vouchers/VoucherRedeem.tsx

```diff
@@ -0,0 +1,298 @@
+                  <RegularText lightest>
+                    {texts.voucher.detailScreen.redeemErrorDescription}
+                  </RegularText>
+                </Wrapper>
+
+                <Wrapper>
+                  <Touchable
+                    onPress={() => {
+                      setIsVisible(false);
+                      setIsRedeemVoucher(false);
+                      setIsExpiredVoucher(false);
+                      setIsChecked(false);
+                    }}
+                  >
+                    <BoldText small center underline lightest>
+                      {texts.voucher.detailScreen.close}
+                    </BoldText>
+                  </Touchable>
+                </Wrapper>
+              </>
+            ) : isRedeemVoucher ? (
+              <>
+                <Wrapper>
+                  <BoldText lightest>{texts.voucher.detailScreen.redeemTitle}</BoldText>
+                </Wrapper>
+
+                <Wrapper style={styles.noPaddingTop}>
+                  <RegularText lightest>{texts.voucher.detailScreen.redeemDescription}</RegularText>
+                </Wrapper>
+
+                <Wrapper style={styles.progressContainer}>
+                  <RegularText style={styles.progressOvertitle} lightest>
+                    {texts.voucher.detailScreen.progressTitle}
+                  </RegularText>
+
+                  <CircularProgress
+                    activeStrokeColor={colors.primary}
+                    inActiveStrokeColor={colors.lighterPrimary}
+                    maxValue={100}
+                    radius={120}
+                    showProgressValue={false}
+                    subtitle={texts.voucher.detailScreen.progressSubtitle}
+                    subtitleColor={colors.surface}
+                    subtitleFontSize={normalize(15)}
+                    title={`${String(minutes).padStart(2, '0')}:${String(seconds).padStart(
+                      2,
+                      '0'
+                    )}`}
+                    titleColor={colors.surface}
+                    titleFontSize={normalize(50)}
+                    titleStyle={styles.progressTitle}
+                    value={((defaultTime - remainingTime) / defaultTime) * 100}
+                  />
+                </Wrapper>
+
+                <Wrapper>
+                  <Touchable
+                    onPress={() => {
+                      setIsVisible(false);
+                      setIsRedeemVoucher(false);
+                      setIsChecked(false);
+                    }}
+                  >
+                    <BoldText small center underline lightest>
+                      {texts.voucher.detailScreen.close}
+                    </BoldText>
+                  </Touchable>
+                </Wrapper>
+              </>
+            ) : (
+              <>
+                <Wrapper>
+                  <BoldText lightest>{texts.voucher.detailScreen.sheetTitle}</BoldText>
```

Comment:
there could be a bit more padding on top, compared to the one on the bottom

Reply:
You're right. I think it will be much better this way. But I think this will apply to all the different views. So I will update the whole component.

Reply:
i do not know which different views you mean exactly, but go forward :)